genesyscloud/resource_genesyscloud_init.go

```diff
@@ -49,15 +49,14 @@ func registerDataSources(l registrar.Registrar) {
 	l.RegisterDataSource("genesyscloud_routing_skill", dataSourceRoutingSkill())
 	l.RegisterDataSource("genesyscloud_routing_skill_group", dataSourceRoutingSkillGroup())
 	l.RegisterDataSource("genesyscloud_routing_email_domain", DataSourceRoutingEmailDomain())
+	l.RegisterDataSource("genesyscloud_routing_utilization_label", dataSourceRoutingUtilizationLabel())
 	l.RegisterDataSource("genesyscloud_routing_wrapupcode", DataSourceRoutingWrapupcode())
-	l.RegisterResource("genesyscloud_routing_wrapupcode", ResourceRoutingWrapupCode())
```

Comment:
I've removed this resource because it was in the middle of the data sources, and wrapupcode resource is already added below.